deploy/operator.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       serviceAccountName: monstorak
       containers:
-      - image: uchapaga/monstorak:39d27f2
+      - image: "quay.io/monstorak/monstorak:3b42010"
```

Comment:
As you have shown, can the image be tagged (instead of the number)...
Is it ok to hard code the image in the yaml file or shall we put a place holder (like "IMAGE_NAME")?

Reply:
Provided Image Tag is a Hash that maps to Git history for easy tracking. Each release tag will be much simpler.
We need to provide an image.